templates/pages/core/get-started/developer-setup.md

```diff
@@ -54,3 +64,183 @@ It requires you to have ran `snapcraft` on your project folder at least once, so
 * Run `snap try` in your project folder
 
 The `snap enable` and `disable` commands will let you stop and restart your snap after you changed its code, if for example, it is providing a service that runs when the snap is installed.
+
+## Installing additional snaps
+
+When developing on your device, it can be useful to add the following snaps to enable specific features.
+
+### pulseaudio
+
+Available on armhf, i386, amd64, arm64.
+
+PulseAudio is a sound server for POSIX and Win32 systems. To record or playback audio files, you need to install the pulseaudio snap.
+
+#### Installing pulseaudio
+
+    sudo snap install pulseaudio
+
+#### Using pulseaudio
+
+Note that when executing the following commands, a warning message can be printed ("Failed to create secure directory..."), it can be ignored.
+
+* List all audio modules and devices:
+
+        sudo pulseaudio.pactl list
+
+* List output ports (sinks):
+
+        sudo pulseaudio.pactl list sinks
+
+* List input ports (sources):
+
+        sudo pulseaudio.pactl list sources
+
+* List audio cards:
+
+        sudo pulseaudio.pactl list cards
```

Comment:
I see that you're trying to keep your indents as a multiple of 4 from the start of the line.

When it comes to bulleted items, I would consider the start of the bulleted item to be 2 spaces in, as in where the above sentence starts from - and then anything inside the bulleted item should be indented from there. So this should be 6 spaces in rather than 8. That makes more sense to me as everything contained within that bullet then looks more visually grouped:

``` markdown
- This is a list item
  
      $ a-code-block

  And some more text

  - A nested list item

- Next item
```

But we don't have an official standard or anything so it's just a suggestion.